src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -1,11 +1,9 @@
 package wolox.training.repositories;
 
 import org.springframework.data.jpa.repository.JpaRepository;
-import org.springframework.stereotype.Repository;
 import wolox.training.models.Book;
 
-@Repository
 public interface BookRepository extends JpaRepository<Book, Long> {
 
-    Book findByAuthor(String author);
+    Book findOneByAuthor(String author);
```

Comment:
Add that the output of this repository is Optional to avoid doing null checks in the future. (Optional<Book>)

I leave a link to have some additional information on the subject:

- [Optionals](https://www.baeldung.com/java-optional)